lib/css/atomic/_stacks-typography.less

```diff
@@ -151,6 +151,26 @@ h1, h2, h3, h4, h5, h6, p {
     overflow: hidden;
 }
 
+.v-truncate-fade-sm {
+    .truncation-fade(0)
+}
+
+.v-truncate-fade-md {
+    .truncation-fade(1)
+}
+
+.v-truncate-fade-lg {
+    .truncation-fade(2)
+}
+
+.truncation-fade(@count) {
+    -webkit-mask-image: linear-gradient(180deg,#000 (pow(2, @count) * 100px - 50px),transparent); 
+    mask-image: linear-gradient(180deg,#000 (pow(2, @count) * 100px - 50px)px,transparent); 
```

Comment:
How's our Legacy Edge 18 support? 😬

Reply:
👌

![image](https://user-images.githubusercontent.com/6742479/109360372-f7291380-783b-11eb-9c24-494be7332979.png)

I haven't tested it myself, but I know a guy that has a Legacy Edge VM set up... 😉